app/router.js

```diff
@@ -7,11 +7,11 @@ const Router = EmberRouter.extend({
 });
 
 Router.map(function() {
-  this.route('users');
+  this.route('users', function() {
+    this.route('current', { path: 'current/:user_id' });
+  });
   this.route('books', { path: '/' });
```

Comment:
any '/' route is the main route so it should be the primary route and all routes should correctly nest under it